entity-management-definitions/src/main/java/eu/europeana/entitymanagement/vocabulary/EntityTypes.java

```diff
@@ -5,6 +5,7 @@
 public enum EntityTypes implements EntityKeyword {
   Organization("Organization", "http://www.europeana.eu/schemas/edm/Organization"),
   Concept("Concept", "https://www.w3.org/2009/08/skos-reference/skos.html#Concept"),
+  ConceptScheme("Scheme", "https://www.w3.org/2009/08/skos-reference/skos.html#ConceptScheme"),
```

Comment:
the entityType is the type of the entity saved in the database and used as type in the serialization of entities. The neme of the EnumEntry can be changed to Scheme for generating the uris